### ISSUE TITLE
Intel Arc A770 - RuntimeError: No CUDA GPUs are available

### ISSUE THREAD

**Issue by Novacane01** [XavierXiao/Dreambooth-Stable-Diffusion#114]
I recently bought the new intel gpu due to the 16gb of VRAM that it has in order to be able to generate the regularization images. However when running stable_txt2img.py I get a runtime error saying that no CUDA GPUs are available. Is it only possible to run this with an Nvidia GPU? I have a 2060 but that only has 8GB of VRAM which isn't enough for the images. Are there any known workarounds?

**Comment by valconius** [comment]
You need vanilla pytorch (which you already have and suffices for NVIDIA) plus an extension specific to intel. (On AMD it is a custom build of pytorch without any extension.) Try these instructions:
- https://towardsdatascience.com/pytorch-stable-diffusion-using-hugging-face-and-intel-arc-77010e9eead6

Intel's webpage:
- https://www.intel.com/content/www/us/en/developer/tools/oneapi/optimization-for-pytorch.html

In short, determine your pytorch version, then download and pip install the relevant extension at: https://intel.github.io/intel-extension-for-pytorch/latest/tutorials/installation.html

I am rather curious to see if this works.

**Comment by runner22k** [comment]
@Novacane01 How is ARC working with dreambooth? is it equally fast to NVIDIA cards comparatively?

**Comment by MangoDragonHub** [comment]
> @Novacane01 How is ARC working with dreambooth? is it equally fast to NVIDIA cards comparatively?

Yeah, I would like to know that too. I'm thinking about buying an ARC card instead of an Nvidia card since it's good for 3D rendering and base Stable Diffusion has Intel oneAPI support.
